webtau-open-api/src/main/java/org/testingisdocumenting/webtau/openapi/OpenApiSpec.java

```diff
@@ -79,44 +67,36 @@ public String getSpecUrl() {
         Request.Method requestMethod = Enum.valueOf(Request.Method.class, method);
 
         ApiOperationMatch apiOperation = apiOperationResolver.findApiOperation(relativePath, requestMethod);
-        if (! apiOperation.isPathFound() || ! apiOperation.isOperationAllowed()) {
+        if (!apiOperation.isPathFound() || !apiOperation.isOperationAllowed()) {
             return Optional.empty();
         }
 
-        return Optional.of(new OpenApiOperation(method,
-                combineWithBasePath(apiOperation.getApiOperation().getApiPath().original())));
+        return Optional.of(new OpenApiOperation(method, apiOperation.getApiOperation().getApiPath().original()));
     }
 
     public Set<String> getDeclaredResponses(OpenApiOperation op) {
         return operationsAndResponses.getOrDefault(op, Collections.emptySet());
     }
 
-    private String combineWithBasePath(String url) {
```

Comment:
@tsiq-karold I haven't found a test that relies on it, wonder if it was somewhere on your side. Also couldn't find an alternative to the base path in new API.

Reply:
I vaguely recall writing this but I cannot remember the specifics.  I don't think I need it.